apps/dcos_l4lb/src/dcos_l4lb_mgr.erl

```diff
@@ -44,6 +44,7 @@
 -type key() :: dcos_l4lb_mesos_poller:key().
 -type backend() :: dcos_l4lb_mesos_poller:backend().
 -type ipport() :: {inet:ip_address(), inet:port_number()}.
+-type namespace() :: term().
```

Comment:
Any reason why we moved from `host | netns()` to a generic`term()`?

Reply:
Yes, please see the comment message of the corresponding commit. There is a mess in types of network namespaces in dcos_l4lb. Some functions expect `netns()` (a.k.a. `#netns{}`), but some other requires `#netns.key`. There is no types in all other modules. So it is my mistake that I found out it while I was testing this patch. `dcos_l4lb_mgr` works with keys, not `netns()`. Key can be any `term()`.